lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -200,6 +200,12 @@ public Map<String, Object> handleRequest(JourneyRequest event, Context context)
                         : buildNotCorrelatedResponse(auditEventUser);
             }
 
+            // Reset identity if reprove is true.
```

Comment:
Should this happen earlier in this function? e.g. What should happen if a user is part way through a F2F journey when the flag is set?

Reply:
Had a chat with @shivanshuit914, yeah the reprove_identity check should come first before the F2F checks